# Marine environmental DNA metabarcoding data analyses practical, 11-13 October 2023

## Structure of the practical
The practical will be divided up by days (1-3). We'll hold the 3rd day for the complete case study 

Each day we will work towards our end goal of recapitulating the results of our chosen studies. But remember, our goal is to learn along the way, not to get to the end. Instead, we take our time on the journey that reaches the final results.

One of the most essential skills in computation biology/informatics is the effective sourcing of reference material. I.e. good googling!

As such throughout the 3/4 days, while you will be given a structure to follow, you will also be asked to work out how to do certain tasks independently. But don't worry, the practical support team will be there to help you when you get stuck. Much of what you're asked to do will be new to you and may feel challenging - that's totally normal.

NU staff will support the trainee with a computer in which all necessary software/tools were installed. However, you can install the required tools on your PC using [THIS](https://github.com/ezzbioinf/RFRE/wiki/eDNA-metabrcoding-utilizing-USEARCH-tool--%E2%80%90-Setup-Tools) guide. Moreover, a docker image based on this GitHub has been created, that you can start it on Gitpod using:
````bash
gitpod.io/#https://github.com/ezzbioinf/RFRE
````
1.4. Practical work (1h).
1.4.1 BARCODE OF LIFE DATA SYSTEM, Retrieving and BIN Database (.....).
1.4.2 Taxonomy (....).
1.4.3 Barcode Gap Summary (....).
1.5. Practical work (2h).
1.5.1 Check for uniqueness and quality of DNA barcoding sequences (.....).
1.5.2 The most common sequence editing problems (.....).
1.5.3 Learning to handle the Sequences and utilizing tools of quality control (.....).
1.5.4 GenBank Submission Troubleshooting: Coding Region Annotation
1.6. Practical work (1h).
1.6.1 Phylogenetic Analysis Steps (.....).
1.6.2 Character-Based Methods, Distance-Based Methods (.....).
1.6.3 Substitution Models and Bootstrap test of Phylogeny (.....).
2. Using DNA barcoding gene in marine conservation and in marine
biodiversity and conservation ( Lecturers names)
2.1. Utilizes Environmental DNA (eDNA) as an effective tool in fish monitoring and species detection,
abundance estimation, pipelines analysis and biodiversity (1 h).
2.2. Methods of isolating eDNA (40 min).
2.3. Methods for labeling DNA samples and preparing DNA metabarcode libraries (40 min).
2.4. Detection of multiple fish species from environmental DNA and other samples using MiFish
metabarcoding (40 min).
2.5. Practical work (3h).
2.5.1 Checking the quality of the FASTQ and trimming the tail(.....).
2.5.2 Assembly of paired-end reads (.....).
2.5.3 Filtering for length and quality (.....).
2.5.4 Pooling and dereplication (.....).
2.5.6 OTU clustering (.....).
2.5.7 Denoising (.....).
2.5.8 Chimera removal (.....).
2.5.9 Remapping (.....).
2.6. Practical work (1h).
2.6.1 MiFish Pipeline (.....).
2.6.2 Graphical user interface methods for analyzing metabarcoding data (.....).
3. Metabarcoding taxonomic assignment (.....).
3.1 Practical work (1h)
3.1.1 Metabarcoding taxonomic assignment on files of Fish environmental DNA (eDNA) (.....).
/
